src/components/buttons/TextButton/examples/TextButtonExample.tsx

```diff
@@ -34,6 +34,16 @@ export class TextButtonExample extends ComponentExampleBase {
 					propName: 'disabled',
 					type: 'boolean',
 				},
+				{
+					options: TextButtonPropFontWeight,
+					defaultValue: TextButtonPropFontWeight.heavy,
+					propName: 'fontWeight',
+					type: 'enum',
+				},
+				{
+					propName: 'fontCasingTransform',
+					type: 'boolean',
+				}
```

Comment:
Is this being used? What for? I'd like to chat about this as well.

Reply:
I think I can get by using `privateOptions` here. I'll test it out and blow this away if needed